content/docs/cml/self-hosted-runners.md

```diff
@@ -83,27 +83,27 @@ you do it your way :)
-| ` --cloud`               | Cloud provider to deploy the runner                                                                             | Choices: `aws`,`azure`                                                                                         |
-| `--cloud-region`         | Region where the instance is deployed.                                                                          | choices: `us-east`,`us-west`, `eu-west`, `eu-north`. Also accepts native cloud regions. Defaults to `us-west`. |
-| `--cloud-type`           | Instance type.                                                                                                  | Choices: `m`, `l`, `xl`. Also supports native types like i.e. `t2.micro`                                       |
-| `--cloud-gpu`            | GPU type.                                                                                                       | Choices: `nogpu`,`k80`,`tesla`                                                                                 |
-| `--cloud-hdd-size`       | HDD size in GB.                                                                                                 | Accepts integer values                                                                                         |
-| `--cloud-ssh-private`    | Your private RSA SSH key. If not provided will be generated by the Terraform-provider-Iterative.                | Accepts string                                                                                                 |
-| `--cloud-spot`           | Request a spot instance                                                                                         | Boolean                                                                                                        |
-| `--cloud-spot-price`     | Spot max price. If not specified it takes current spot bidding pricing.                                         | Default: `-1`                                                                                                  |
-| `--cloud-startup-script` | Run the script provided in-line during the instance initialization. Expects a Base64-encoded Linux shell script | String                                                                                                         |
-| `-h`                     | Show help                                                                                                       | Boolean                                                                                                        |
+| Argument                 | Type                                                                  | Default                                                    | Description                                                                                                                  |
```

Comment:
So are these "arguments"? Or flags/options? We call them Options in DVC cmd refs e.g. https://dvc.org/doc/command-reference/add — not saying that's correct though, I'm actually wondering what's the best terminology for this...

For now maybe call this section **Options** and this column **Name** (i.e. option name), avoiding the "argument" term.

Reply:
> maybe call this section Options and this column Name

Then again it could keep **Arguments** for the title and **Option** for the column, kind of like in https://docs.microsoft.com/en-us/windows/terminal/command-line-arguments?tabs=windows#options-and-commands. Up to you

Reply:
I guess I'd avoid "argument" for title or column since some options/flags accept arguments of their own.

Reply:
As per [POSIX.1-2017](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html#tag_12_01), commands take **options** that may require **option arguments** and anonymous **operands**.  Definitely, it makes sense to rename both the column and the heading.

Reply:
OK! Can you please update the title to "Options" then? (I can't from GH.) Here are the col renames I rec:

```suggestion
| Name                     | Type                                                                  | Default argument                                           | Description                                                                                                                  |
```

Reply:
Resolved with https://github.com/iterative/dvc.org/pull/2321/commits/ca804af226acd6100e23d25430d39b87277ef6f8